### ISSUE TITLE
Could not establish connection (Error 403 ) when creating new project

### ISSUE THREAD

**Issue by urbandroid** [emiloberg/liferay-ddmtool#15]
with  liferay-ce-portal-7.0-ga5 ,  ddm prints `Could not establish connection (Error 403 )` to terminal even though authentication credentials are correct when i try to debug i found that problem is not about authentication.

To debug when i add

```
console.log('\n'+ 'Request:  \n');
			console.log(res.req);
```  
to after  https://github.com/emiloberg/liferay-ddmtool/blob/35593d809bb81208be14b2dcfeeb15471b8fdf76/lib/getData.js#L37  

it throw a message which contains 

> 'com.liferay.portal.kernel.jsonwebservice.NoSuchJSONWebServiceException: No JSON web service action with path /classname/fetch-class-name-id and method null for null'

after that i checked the http://localhost:8080/api/jsonws and server was telling the truth only services contains classname was `fetch-by-class-name-id` and
`fetch-class-name` no `fetch-class-name-id ` i'm assuming this is due to difference between rest api of liferay versions.

in conclusion ddm is not usable with liferay-ce-portal-7.0-ga5
